sdk/react-native/src/withDVCProvider.ts

```diff
@@ -0,0 +1,17 @@
+import { withDVCProvider as reactWithDVCProvider } from '@devcycle/devcycle-react-sdk'
+import { ProviderConfig } from './types'
+import { ReactNativeStore } from './ReactNativeCacheStore'
+
+export default function withDVCProvider(config: ProviderConfig) {
```

Comment:
you should just make this be the same type as the react provider rather than redefining ProviderConfig here

Reply:
will need to export the type from the React SDK as it currently do that

Reply:
`typeof reactWithDVCProvider`